Advertise first Knative Virtual Office Hour 2022/12/09 (#5341)
knative/docs: blog/docs/events/virtual-office-hour-12-2022.md

> ![Knative Booth](images/virtual-office-hour-12-2022-with-text.jpg)
> # First 🌱 Knative Virtual Office Hour

Reviewer: ```suggestion
# First 🌱 Knative Virtual Office Hour

![Knative Booth](images/virtual-office-hour-12-2022-with-text.jpg)
```

I think due to how mkdocs renders the page, the first line needs to be the title, otherwise it uses the filename on the left-side menu.

Author: Moved to safe location - hopefully mkdocs will be happy

**Date: 2022-12-01**

**Authors: [Alek Slominski](https://aslom.net/), [Evan Anderson](https://off-by-one.dev/), [Lance Ball](https://twitter.com/lanceball), and Paul Schweigert**

![Knative Booth](images/virtual-office-hour-12-2022-with-text.jpg)

We think that there should be an easy way for new Knative users and contributors to get started with our project. Holding virtual office hours for questions and help with using or contributing to Knative should help to get started with using code and contributing.

During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed there are people interested in using or contributing to Knative. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is asynchronous and requires joining yet another Slack, then reading channels, and figuring where to ask your questions. Driving your own engagement can be a barrier to participating in the community.

Therefore, we are organizing the first Knative Virtual Office Hour in 2022.

> **When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)

Reviewer 2: ```suggestion
**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
<a target="_blank" href="https://calendar.google.com/calendar/event?action=TEMPLATE&amp;tmeid=NXFjbzVjM3NqZTU1aHNoYmtkbjA5am1uOW4ga25hdGl2ZS50ZWFtXzlxODNiZzA3cXM1YjlycnNscDVqb3I0bDZzQGc&amp;tmsrc=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com"><img border="0" alt="Add to Google Calendar" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>
```

Reviewer 2: (I created a calendar entry)

Author: :+1:

> **When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
> **How to join:** Zoom link will be posted in Slack and knative-user mailing list

Reviewer: ```suggestion
**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)

**How to join:** Zoom link will be posted in Slack and knative-user mailing list
```

Author: done

During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed there are people interested in using or contributing to Knative. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is asynchronous and requires joining yet another Slack, then reading channels, and figuring where to ask your questions. Driving your own engagement can be a barrier to participating in the community.

Therefore, we are organizing the first Knative Virtual Office Hour in 2022.

**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
<a target="_blank" href="https://calendar.google.com/calendar/event?action=TEMPLATE&amp;tmeid=NXFjbzVjM3NqZTU1aHNoYmtkbjA5am1uOW4ga25hdGl2ZS50ZWFtXzlxODNiZzA3cXM1YjlycnNscDVqb3I0bDZzQGc&amp;tmsrc=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com"><img border="0" alt="Add to Google Calendar" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>

> **How to join:** Zoom: https://zoom.us/j/94310596244

Reviewer: My bad, I gave you a non-clickable link :cry: 

```suggestion
**How to join:** Zoom: [https://zoom.us/j/94310596244](https://zoom.us/j/94310596244)
```

> **How to join:** Zoom link will be posted in Slack and knative-user mailing list

Reviewer: ```suggestion
**How to join:** Zoom: https://zoom.us/j/94310596244
```

# First 🌱 Knative Virtual Office Hour

**Date: 2022-12-01**

**Authors: [Alek Slominski](https://aslom.net/), [Evan Anderson](https://off-by-one.dev/), [Lance Ball](https://twitter.com/lanceball), and Paul Schweigert**

> We think that there should be an easy way for new Knative users and contributors to get started with our project. Holding virtual office hours for questions and help with using or contributing to Knative should help to get started with using code and contributing.

Reviewer 2: Can we add a splash image, like https://unsplash.com/photos/Q0AM87PsYkE ?

Author: Maybe one of photoa from KubeCon with Knative in background form KnativeCon or other meeting?

Reviewer 2: 👍 

That's a great idea!

> During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed that there are people interested in using Knative or contributing to it. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is very asynchronous and joining yet another Slack and then reading channels, figuring where to ask your questions, and driving your own engagement can be a barrier to participating in the community.

Reviewer 3: Just a little bit of wordsmithing:
```suggestion
During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed there are people interested in using or contributing to Knative. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is asynchronous and requires joining yet another Slack, then reading channels, and figuring where to ask your questions. Driving your own engagement can be a barrier to participating in the community.
```

> Therefore, we are organizing the first Knative Virtual Office Hour / Contrib Fest in 2022.

Reviewer 3: Agree with @evankanderson that the focus should be on the users, and less so contributors.
```suggestion
Therefore, we are organizing the first Knative Virtual Office Hour in 2022.
```

**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)

> **How to join:** Zoom link will be posted in Slack and knative-user mailing list

Reviewer 2: We should go back and update this when we have the calendar, but there's no reason not to post the blog post now.

Author: :+1:

**Who should join?**

- You are curious about Knative and you are wondering where to start

> - You have great idea for Knative but you do not want where to start

Reviewer 3: ```suggestion
- You have great idea for Knative but you do not know where or how to share it
```

> - You have great idea for Knative but you do not know where or how to share it

Reviewer 2: ```suggestion
- You have questions about how to build something using Knative
- You have great idea for Knative but you do not know where or how to share it
- You want to meet other people using Knative
```

- You want to get to know how Knative works inside and want to contribute to open source project

> **Possible topics for Contib fest:**
> 
> - See [slides form KubeCon about Knative ContribFest](https://docs.google.com/presentation/d/1hnPHtbjLs3KMe-N62NhQeVtjH-p-obyI/edit?usp=sharing&ouid=112051875307901839973&rtpof=true&sd=true)
> - Browse good first issues:
>   - [client](https://github.com/knative/client/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
>   - [docs](https://github.com/knative/docs/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
>   - [eventing](https://github.com/knative/eventing/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
>   - [func](https://github.com/knative/func/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
>   - [serving](https://github.com/knative/serving/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
> - Browse eventing sources in knative-sandbox to see what issues could be fixed of features added such as
>   - [eventing-couchdb](https://github.com/knative-sandbox/eventing-couchdb)
>   - [eventing-github](https://github.com/knative-sandbox/eventing-couchdb)
>   - [eventing-gitlab](https://github.com/knative-sandbox/eventing-couchdb)
>   - [eventing-kogito](https://github.com/knative-sandbox/eventing-couchdb)
>   - [eventing-redis](https://github.com/knative-sandbox/eventing-couchdb)
>   - Other [eventing sources and sandbox projects](https://github.com/knative-sandbox/)

Reviewer 2: ```suggestion
```

I'm not sure we want to link the Contrib fest part here past the slides.

Instead, I added a couple more items for "who should join".

Author: OK with me - we can see if there is need for virtual contrib fest in future.

---
title: "First 🌱 Knative Virtual Office Hour / Contrib Fest 2022/12/09"
type: "blog"
date: 2022-12-01
linkTitle: "First Knative Virtual Office Hour / Contrib Fest 2022/12/09"

> description: "Join the Zoom Knative First Knative Virtual Office Hour / Contrib Fest on 2022/12/09"

Reviewer 2: I might call this "Virtual Office Hour" and drop the "Contrib Fest" part. I think having "open for new contributors" in the text body is okay, but I think having end-users show up just to ask questions or chat should be explicitly our goal, and adding "ContribFest" adds a bunch of pressure to become a contributor.

> ---

Reviewer: ```suggestion
**Authors: [Alek Slominski](https://aslom.net/), [Evan Anderson](https://off-by-one.dev/), and Paul Schweigert**

**Date: 2022-12-01**

# First 🌱 Knative Virtual Office Hour / Contrib Fest 2022/12/09
```

this is the new format we're using for blog headers that we switched to a little bit ago

Author: Updated to the new blog format.

Let make separate PR to docs/blog/config/nav.yml and docs/blog/docs/index.md when this PR is merged?

Reviewer: I'd do at least the nav.yaml one in this PR, otherwise the blog post won't be visible.

## First 🌱 Knative Virtual Office Hour / Contrib Fest

We think that there should be an easy way for new Knative users and contributors to get started with our project. Holding virtual office hours for questions and help with using or contributing to Knative should help to get started with using code and contributing.

> During the recent KubeCon NA 2022 in Detroit we had Knative Day and ContribFest that showed that there are people interested in using Knative or contributing to it. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is very asynchronous and joining yet another Slack and then reading channels, figuring where to ask your questions, etc. feels very daunting,

Reviewer 2: ```suggestion
During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed that there are people interested in using Knative or contributing to it. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is very asynchronous and joining yet another Slack and then reading channels, figuring where to ask your questions, and driving your own engagement can be a barrier to participating in the community.
```

Therefore, we are organizing the first Knative Virtual Office Hour / Contrib Fest in 2022.

**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
**How to join:** Zoom link will be posted in Slack and knative-user mailing list

**Who should join?**

- You are curious about Knative and you are wondering where to start

> - You would like to contribute to Knative but do not know where to start

Reviewer 2: This feels like a duplicate of the last one.

Maybe:

> You are trying to use Knative and trying to figure out how to do a specific thing with it.

> - You have problems with Knative and asking in Slack or Github issues did not get you what you wanted

Reviewer 2: I'd remove the "asking in Slack or GitHub", that feels like it might shame people into not attending. We should steer them _after_ they show up, rather than making them feel like they can't show up. 😁

> - You have great idea for Knative but yo do not want where to start

Reviewer 2: ```suggestion
- You have great idea for Knative but you do not want where to start
```

- Browse good first issues:
  - [client](https://github.com/knative/client/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
  - [docs](https://github.com/knative/docs/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
  - [eventing](https://github.com/knative/eventing/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
  - [func](https://github.com/knative/func/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
  - [serving](https://github.com/knative/serving/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
- Browse eventing sources in knative-sandbox to see what issues could be fixed of features added such as
  - [eventing-couchdb](https://github.com/knative-sandbox/eventing-couchdb)
  - [eventing-github](https://github.com/knative-sandbox/eventing-couchdb)
  - [eventing-gitlab](https://github.com/knative-sandbox/eventing-couchdb)
  - [eventing-kogito](https://github.com/knative-sandbox/eventing-couchdb)
  - [eventing-redis](https://github.com/knative-sandbox/eventing-couchdb)
  - Other [eventing sources and sandbox projects](https://github.com/knative-sandbox/)

No prior knowledge of Knative is required.

> Hope to see you there!

Reviewer 2: I think the feedback from the Maintainer Meetup was that we didn't have a community <-> contributors dialogue; I'd like to focus this more on that dialogue than on immediately moving folks into contributing.

Author: Yes. When we have the blog post published I think we should send email to knative-users to invite them to the office hour for dialogue and for us to listen.
